src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirer.cs

```diff
@@ -83,6 +83,7 @@ async Task<AcquireTokenResult> ITokenAcquirer.GetTokenForAppAsync(string scope,
                 UserFlow = tokenAcquisitionOptions.UserFlow,
                 PopPublicKey = tokenAcquisitionOptions.PopPublicKey,
                 PopClaim = tokenAcquisitionOptions.PopClaim,
+                ExtraParameters = tokenAcquisitionOptions.ExtraParameters,
```

Comment:
This isn't used.

Reply:
It is used, this is used by extensions. See the SAL PR.

Reply:
Checking...

Reply:
Got it, yes, thanks for point this out @jennyf19